jaxws-ri/rt/src/main/java/com/sun/xml/ws/util/xml/XMLStreamWriterFilter.java

```diff
@@ -53,6 +53,26 @@ public void writeStartDocument() throws XMLStreamException {
         writer.writeStartDocument();
     }
 
+    private boolean isWhiteSpace(char c){
+        // all char points between 0 and 31 excluding the allowed white spaces (9=TAB, 10=LF, 13=CR)
+        if (c >= 0 && c<=31 && c != 9 && c != 10 && c != 13)
+                return true;
+
+        return false;
+    }
```

Comment:
It's bad name for that method.
What is the value of and what is expected to be for
```java
isWhiteSpace(' ');
```
?

Replace comment with better name.
One can be confused by different results of [Character.isWhitespace(char)](https://docs.oracle.com/javase/7/docs/api/java/lang/Character.html#isWhitespace(char))

Reply:
I can change the method name to "isUnusualChar"
Will it be fine
The value of isWhiteSpace(' ') above will be false as ASCII value of ' ' is 32 
which is expected since it will not transform ' '

Reply:
> I can change the method name to "isUnusualChar"

IMO - it would be better.